### DISCUSSION TITLE
[FEATURE] LISTING_OFFER_ONLY for vendors

### DISCUSSION THREAD

**Discussion by lpongetti** [amzn/selling-partner-api-models#2803] [General]
Why is LISTING_OFFER_ONLY not supported for vendors?
I'm forced to enter all the data for each catalog item even if it already exists...

![image](https://github.com/amzn/selling-partner-api-docs/assets/6831116/1465b7d5-e83b-4cdd-bc44-5cf294c75750)

**Comment by rugved1991** [comment]
Hi @lpongetti,

Vendors own their listings so they should be able to create a listing with all the information. You can choose to create a new listing with or without sales terms. You can use LISTING_PRODUCT_ONLY to provide only product facts and not the sales terms. 
But, if you are planning to edit an existing item then you should be using the PATCH operation instead of a PUT to only update the attributes that you need to change. 

LISTING_OFFER_ONLY requirement is very seller specific as multiple sellers can create an offer on an item which is owned by a different entity. For such items you dont need to fill out the entire product information as it is already present in Amazon catalog.

Best,
Rugved
Solutions Architect, SP API

**Comment by lpongetti** [comment]
Hi @rugved1991, thanks for the reply,
There are too many properties required to set to insert a listings and it's impossible to do a automatic insert. 
We are tring to replace EDI insert method to use metadata of another vendor. Now with this api is impossile for us.
There's a way to do an import via API with EDI info's?
